benchmarks/models.py

```diff
@@ -19,6 +19,10 @@
 
 class ManifestReduced(models.Model):
     hash = models.CharField(primary_key=True, max_length=40)
+    created_at = models.DateTimeField(default=timezone.now)
```

Comment:
well, I didn't include the removal of this data because we also didn't include  the removal of the other data (Result + ResultData + TestJob) which we also deleted by hand ... so the crash with a unclean database is due to an operation error of ours